server/apis/v1/handler.go

```diff
@@ -1335,3 +1362,23 @@ func (h *handler) getDaemonClient(ns, pipeline string) (daemonclient.DaemonClien
 		return dClient, nil
 	}
 }
+
+func (h *handler) getMonoVertexDaemonClient(ns, mvtName string) (mvtdaemonclient.MonoVertexDaemonClient, error) {
+	if mvtDaemonClient, ok := h.mvtDaemonClientsCache.Get(daemonSvcAddress(ns, mvtName)); !ok {
```

Comment:
should have a different function for mvtx daemon svc address, they are different.

Reply:
thanks, addressed.